analysis/src/Xform.ml

```diff
@@ -176,7 +192,23 @@ module AddBracesToFn = struct
       let newText = printStructureItem ~range newStructureItem in
       let codeAction =
         CodeActions.make ~title:"Add braces to function" ~kind:RefactorRewrite
-          ~uri:path ~newText ~range
+          ~command:None
+          ~edit:
+            (Some
```

Comment:
this part is repeated multiple times, can you factor that out in its own function?

Reply:
https://github.com/rescript-lang/rescript-vscode/pull/767/commits/817460413b27be9fe7c0d89ebbea733216e4efa8